Example/iOS Example/WillowConfiguration.swift

```diff
@@ -34,103 +34,108 @@ struct WillowConfiguration {
         let name: String
 
-        init(emoji: String, name: String) {
-            self.emoji = emoji
+        init(prefix: String, name: String) {
+            self.prefix = prefix
             self.name = name
         }
 
-        func modifyMessage(_ message: String, with: LogLevel) -> String {
-            return emoji + " [" + name + "] => " + message
+        func modifyMessage(_ message: String, with logLevel: LogLevel) -> String {
+            switch logLevel {
+            case .warn:  return "🚨🚨🚨 [\(name)] => \(message)"
+            case .error: return "💣💥💣💥 [\(name)] => \(message)"
+            default:     return "\(prefix) [\(name)] => \(message)"
+            }
         }
     }
 
-    private struct WarningPrefixModifier: LogMessageModifier {
-        func modifyMessage(_ message: String, with: LogLevel) -> String {
-            return "🚨🚨🚨 \(message)"
+    // MARK: Writers
+
+    private class ServiceWriter: LogWriter {
+        func writeMessage(_ message: String, logLevel: LogLevel) {
+            // Send the message as-is to our external logging service
+            let attributes: [String: Any] = ["LogLevel": logLevel.description]
+
+             ServiceSDK.recordBreadcrumb(message, attributes: attributes)
         }
-    }
 
-    private struct ErrorPrefixModifier: LogMessageModifier {
-        func modifyMessage(_ message: String, with: LogLevel) -> String {
-            return "💣💥💣💥 \(message)"
+        func writeMessage(_ message: Message, logLevel: LogLevel) {
+            // Send the message as-is to our external logging service
+            var attributes = message.attributes
+            attributes["LogLevel"] = logLevel.description
+
+            ServiceSDK.recordBreadcrumb(message.name, attributes: attributes)
         }
     }
 
     // MARK: - Configure
 
-    static func configure(
-        appLogLevels: LogLevel = [.debug, .info, .event],
-        databaseLogLevels: LogLevel = [.sql, .debug, .info, .event],
-        webServicesLogLevels: LogLevel = [.debug, .info, .event],
-        asynchronous: Bool = false)
+    static func configure()
     {
-        let writers: [LogLevel: [LogMessageWriter]] = [.all: [ConsoleWriter()]]
-        let executionMethod: LoggerConfiguration.ExecutionMethod
-
-        if asynchronous {
-            executionMethod = .synchronous(lock: NSRecursiveLock())
-        } else {
-            executionMethod = .asynchronous(
-                queue: DispatchQueue(label: "com.nike.example.logger", qos: .utility)
-            )
-        }
-
-        log = configureLogger(
-            emoji: "🌳🌳🌳",
+        #if DEBUG
+            // A debug build of the application would probably log at a higher level than a release version.
+            // Also, synchronous logging can be beneficial in debug mode so that log statements emit as you step through the code.
+            let appLogLevels: LogLevel = [.all]
+            let databaseLogLevels: LogLevel = [LogLevel.Database.sql, .all]
```

Comment:
`.all` automatically includes the `.sql` log level since it's a bit mask of 1's. Would it be better to just enable the log levels individually in the example? Such as:

```swift
let databaseLogLevels: LogLevel = [LogLevel.Database.sql, .debug, .info, .event]
```